lib/matplotlib/backends/backend_agg.py

```diff
@@ -563,23 +563,26 @@ def print_jpg(self, filename_or_obj, *args, dryrun=False, pil_kwargs=None,
             `PIL.Image.save` when saving the figure.  These take precedence
             over *quality*, *optimize* and *progressive*.
         """
-        FigureCanvasAgg.draw(self)
-        if dryrun:
-            return
         # The image is "pasted" onto a white background image to safely
```

Comment:
The comment needs to be adapted.

Reply:
well, this is still pasting the figure over a white background.